main.c

```diff
@@ -144,13 +147,16 @@ int main(int argc, char *argv[]) {
         { "ignore",   no_argument      , 0, 'i' },
         { "help",     no_argument      , 0, 'h' },
         { "verbose",  no_argument      , 0, 'v' },
+        { "version",  no_argument      , 0, 'V' },
         { "brief",    no_argument      , 0, 'b' },
         { 0,          0,                 0,  0  }
     };
 
     int opt = 0;
     int long_index = 0;
-    while ((opt = getopt_long_only(argc, argv, "l:u:m:L:U:ihvb", long_options, &long_index)) != -1) {
+	program_name = basename(argv[0]);
```

Comment:
Probably doesn't need to be indented that far.
```suggestion
    program_name = basename(argv[0]);
```

Reply:
Good catch, thanks!